Sources/StreamChatUI/CommonViews/CloseButton/ChatCloseButton.swift

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// Button for closing, dismissing or clearing information.
+open class ChatCloseButton: _Button, AppearanceProvider {
```

Comment:
Something that is not clear yet for me is: Should all our views have `Chat` prefix, or only the ones that use the LLC models, example `ChatMessageInputView` ? If this is the case, then all these button components prefixes should be removed

Reply:
@nuno-vieira I don't think we need `Chat` prefix for all components. I think the _default_ should be no prefix. The `Chat` prefix should be used only for components where it's expected users will interact with them a lot and they have a too generic name. `User` -> `ChatUser` for example.

Reply:
I think we should treat all the components the same, we don't know what the users will or won't use. As long as they can import only a few specific views, the Stream views in the client's codebase should be easily recognisable.

Reply:
@VojtaStavik Jakub has a good point, if we prefix all views with `Chat`, I can see 3 advantages: 
- The views are easily recognizable as a view from the SDK
- The view names are consistent
- When naming a new view, you don't need to think about if it needs the `Chat` prefix or not